Chain config per node

felipemadero opened the pull request:
Closes https://github.com/ava-labs/avalanche-cli/issues/435

dasconnor reviewed (commented):
Can we add an E2E test and some documentation on how to use this feature?

felipemadero commented:
> Can we add an E2E test and some documentation on how to use this feature?

added a comment into
https://github.com/ava-labs/avalanche-cli/pull/508#pullrequestreview-1214457072
added
https://github.com/ava-labs/avalanche-docs/pull/1130

dasconnor commented:
maybe give @holisticode a chance to look over this before we merge
